docs/general/events.md

```diff
@@ -17,74 +17,31 @@ network connection severed, etc.
 
 In OpenTelemetry, Events are implemented as a specific type of [`LogRecord`](https://github.com/open-telemetry/opentelemetry-specification/tree/v1.44.0/specification/logs/api.md) that conforms to the conventions included here.
 
-In addition to a required name, an Event may contain a _payload_ (body) of any type permitted
-by the [LogRecord body](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/logs/data-model.md#field-body).
-In its implementation, the Event _payload_ (body) will constitute the `Body` of the `LogRecord`.
-Like all other OpenTelemetry signals, an Event has optional attribute metadata that helps describe
-the event context.
-
-Over time, some Events will be specified by OpenTelemetry and will have documented payload structure,
-field semantics, and stability and requirement levels. Other events may be user-defined and carry
-bespoke user semantics. When an Event name exists in the semantic conventions, its _payload_
-structure and semantics will also be defined.
+OpenTelemetry Semantic Conventions that define events SHOULD document the event name along
+with attributes and the type of the body if any.
 
 ## General event semantics
 
 * An event MUST have an [Event name property](https://github.com/open-telemetry/opentelemetry-specification/tree/v1.44.0/specification/logs/data-model.md#field-eventname)
-  that uniquely identifies the event. Event names are subject to the [Naming guidelines](/docs/general/naming.md).
-* Event MAY have [standard](https://github.com/open-telemetry/opentelemetry-specification/tree/v1.44.0/specification/common#attribute)
+  that uniquely identifies the event. Event names are subject to the [Naming guidelines](/docs/general/naming.md). Event name SHOULD be of a low-cardinality.
+* Events MAY have [extended](https://github.com/open-telemetry/opentelemetry-specification/blob/v1.44.0/specification/logs/data-model.md#field-attributes)
   attributes that provide additional context about the event.
```

Comment:
I suggest not using the "extended" term as it may confuse the users.
We do not have such term in the specification.
Moreover, the hyperlink references the log attributes which is correct.

```suggestion
* Events MAY have [attributes](https://github.com/open-telemetry/opentelemetry-specification/blob/v1.44.0/specification/logs/data-model.md#field-attributes)
  that provide additional context about the event.
```